src/file-io.c

```diff
@@ -611,10 +381,7 @@ void gtt_load_config(void)
     }
     g_free(s);
 
-    /* OK, try to load the oldest file format */
-    project_list_load_old();
     config_data_url = XML_DATA_FILENAME;
```

Comment:
Initially I had dropped this assignment of `config_data_url` too. But in case of the very first start of GnoTime it seems to reach this line and lead to undefined behaviour if `config_data_url` is not set.